BUILD

```diff
@@ -0,0 +1,13 @@
+# Copyright Istio Authors.
```

Comment:
Should copyright be Istio Authors or something else? https://github.com/istio-ecosystem/admiral is not attributed to Istio Authors, for example.

Reply:
Any suggestion? Looks like admiral does not specify copyright for the code they wrote. I can just remove them as well.

Reply:
Maybe remove it for now? I am not sure about this org policy.

Reply:
Done.